.gitignore

```diff
@@ -17,6 +17,8 @@ develop-eggs
 .installed.cfg
 lib
 lib64
+share
```

Comment:
If this is not directly related to the streaming API, please submit changes into a pull request in the main repo.